.github/workflows/test.yml

```diff
@@ -14,6 +14,19 @@ defaults:
   run:
     shell: bash
 jobs:
+  e2e-test:
+    runs-on: ubuntu-20.04
+    name: "Sentry self-hosted end-to-end tests"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: self-hosted
+      - name: End to end tests
+        uses: getsentry/action-self-hosted-e2e-tests@a279745f971086de369c9573d59c332f24af9d5f
```

Comment:
I'm using a commit sha because I will likely be iterating on the action, and I don't feel like making a tag yet is right.